### ISSUE TITLE
[Accessibility] My Store Quick TalkBack Audit

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#1940]
## Is your feature request related to a problem? Please describe.
This is just a quick TalkBack audit for My Store. 

<img src="https://user-images.githubusercontent.com/198826/74456924-c0f13f80-4e44-11ea-940c-5774aa584de7.png" width="240">

## Describe the solution you'd like

Here is a quick GIF of how navigation with TalkBack works right now.

<img src="https://user-images.githubusercontent.com/198826/74457057-feee6380-4e44-11ea-9f6a-5fb576ab3769.gif" width="320">

### Stats

The Visitors, Orders, and Revenue stats labels and their values are separately accessible. With this, there are at least 6 accessible elements on the screen, which can make it convoluted. Also, focusing on an individual number would not be an ideal experience because you would not know right away what the number is for. 

The experience could probably be improved if the elements are grouped into one: 

<img src="https://user-images.githubusercontent.com/198826/74457569-abc8e080-4e45-11ea-83aa-6e4eda1675fe.png" width="320">

When the user focuses on that single element, TalkBack can read it as:

> 10 visitors. 190 orders. 1,291 revenue. Updated moments ago. 

### Empty View

The empty view has at least 3 accessible elements. The title and the description can be grouped into one accessible element.

<img src="https://user-images.githubusercontent.com/198826/74457983-3f9aac80-4e46-11ea-9737-cbb2c2fbcad0.png" width="320">

---

**Comment by JorgeMucientes** [comment]
Closing this as part of a maintenance pass on the GitHub issue backlog. This issue matched the auto-close criteria for this pass: it has not been updated since 2020-02-13, is unassigned, is not labeled as a bug or high/medium priority, and does not appear to involve POS, payments, login, data loss, crash, security, vulnerability, auth, or fraud.

If this is still relevant, please comment with current details and we can reopen it or move it into Linear for triage.
